Description: Disable benches that use criterion so the rest of the testsuite can run.
Author: Peter Michael Green <plugwash@debian.org>

Index: zvariant/Cargo.toml
===================================================================
--- zvariant.orig/Cargo.toml
+++ zvariant/Cargo.toml
@@ -53,8 +53,6 @@ version = "1.1.0"
 
 [dependencies.zvariant_derive]
 version = "=2.10.0"
-[dev-dependencies.criterion]
-version = "0.3"
 
 [dev-dependencies.rand]
 version = "0.8.3"
Index: zvariant/benches/benchmarks.rs
===================================================================
--- zvariant.orig/benches/benchmarks.rs
+++ zvariant/benches/benchmarks.rs
@@ -2,14 +2,14 @@ use byteorder::LE;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
-use criterion::{black_box, criterion_group, criterion_main, Criterion};
+//use criterion::{black_box, criterion_group, criterion_main, Criterion};
 
 use zvariant::{
     from_slice_for_signature, to_bytes_for_signature, EncodingContext as Context, Type, Value,
 };
 use zvariant_derive::Type;
 
-fn fixed_size_array(c: &mut Criterion) {
+/*fn fixed_size_array(c: &mut Criterion) {
     let ay = vec![77u8; 100_000];
     let ctxt = Context::<LE>::new_dbus(0);
     let signature = Vec::<u8>::signature();
@@ -26,9 +26,9 @@ fn fixed_size_array(c: &mut Criterion) {
                     .unwrap();
         })
     });
-}
+}*/
 
-fn big_array_ser_and_de(c: &mut Criterion) {
+/*fn big_array_ser_and_de(c: &mut Criterion) {
     #[derive(Deserialize, Serialize, Type, PartialEq, Debug, Clone)]
     struct ZVField<'f> {
         int2: u64,
@@ -123,7 +123,11 @@ fn big_array_ser_and_de(c: &mut Criterio
             black_box(s);
         })
     });
-}
+}*/
 
-criterion_group!(benches, big_array_ser_and_de, fixed_size_array);
-criterion_main!(benches);
+//criterion_group!(benches, big_array_ser_and_de, fixed_size_array);
+//criterion_main!(benches);
+
+fn main() {
+    println!("Benchmarks disabled because criterion is not in Debian");
+}
\ No newline at end of file
